lib/service/service.go

```diff
@@ -923,15 +923,23 @@ func initExternalLog(auditConfig services.AuditConfig) (events.IAuditLog, error)
 		case dynamo.GetName():
 			hasNonFileLog = true
 			cfg := dynamoevents.Config{
-				Tablename: uri.Host,
-				Region:    auditConfig.Region,
+				Tablename:               uri.Host,
+				Region:                  auditConfig.Region,
+				EnableContinuousBackups: auditConfig.EnableContinuousBackups,
+				EnableAutoScaling:       auditConfig.EnableAutoScaling,
+				ReadMinCapacity:         int(auditConfig.ReadMinCapacity),
```

Comment:
Just curious, would it make sense to make `ReadMinCapacity` and other new fields be of a consistent type everywhere - either `int` or `int64` and avoid conversions (and potentially int overflow on 32 bit systems)? Seems like it's `int64` in the proto file but `int` in the config struct. I would prob. go with the type that AWS Go SDK accepts. Disregard if there's a technical reason for this, or too much refactoring required.